src/plugins/visualizations/public/wizard/type_selection/type_selection.tsx

```diff
@@ -222,7 +222,7 @@ class TypeSelection extends React.Component<TypeSelectionProps, TypeSelectionSta
   private renderVisType = (visType: VisTypeListEntry) => {
     let stage = {};
     let highlightMsg;
-    if (!isVisTypeAlias(visType.type) && visType.type.stage === 'experimental') {
+    if (visType.type.stage === 'experimental') {
```

Comment:
It's a little tricky to make out what the original intent was here. But is `type` a required property, or do we need `visType?.type.stage`?

Reply:
based on the Typescript type, it looks like its a required prop. I also did a sanity test with the code and it looks okay.